src/main/asciidoc/index.adoc

```diff
@@ -286,3 +286,45 @@ dependencies {
  compile "com.hazelcast:hazelcast:ENTER_YOUR_VERSION_HERE"
 }
 ----
+
+== Kubernetes and Rolling updates
```

Comment:
Would you mind to add a _Configuring for Kubernetes_ section similar to what we have in `vertx-infinispan` and then make this one a subsection?

We had a few questions about discovery on Kubernetes in the past, since you know how it works it's a good opportunity to tackle this issue as well.